gnucash/register/ledger-core/split-register-model.c

```diff
@@ -2007,6 +2009,10 @@ gnc_split_register_confirm (VirtualLocation virt_loc, gpointer user_data)
     trans = xaccSplitGetParent (split);
     if (xaccTransWarnReadOnly(trans))
         return FALSE;
+
+    if (xaccTransHasReconciledSplits (trans) == FALSE)
```

Comment:
Nitpick: while correct, the preferred style is `if (!xaccTransHasReconciledSplits (trans))`